todo/src/actions/actions.js

```diff
@@ -0,0 +1,24 @@
+export const ADD_TODO = "ADD_TODO";
+export const TOGGLE_COMPLETED = "TOGGLE_COMPLETED";
+export const REMOVE_TODO = "REMOVE_TODO";
+
+export function addTodo(todo) {
+  return {
+    type: ADD_TODO,
+    payload: todo
+  };
+}
+
+export function toggleCompleted(index) {
+  return {
+    type: TOGGLE_COMPLETED,
+    payload: index
```

Comment:
What if you couldn't pass the index to your reducer in your action object? How would you handle your reducer?